components/common/Header.jsx

```diff
@@ -17,23 +18,59 @@ const Nav = styled.nav`
   height: 52px;
   z-index: 9997;
   position: sticky;
-  transition: background-color 0.5s cubic-bezier(0.28, 0.11, 0.32, 1);
-  background-color: var(--material-background-color);
-  backdrop-filter: var(--material-filters);
-  box-shadow: 0 1px 0 0 var(--material-separator-color);
   * {
     box-sizing: content-box;
   }
+  @media ${mediaQueries.sm} {
+    height: 48px;
+  }
 `;
 const Wrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: auto;
+  min-height: 100%;
+  z-index: 1;
+  transition: background-color 0.5s cubic-bezier(0.28, 0.11, 0.32, 1);
+  background-color: var(--material-background-color);
+  backdrop-filter: var(--material-filters);
+  box-shadow: 0 1px 0 0 var(--material-separator-color);
+  .menu-open & {
+    background-color: var(--thick-material-background-color);
+  }
 `;
 const HeaderContainer = styled(Container)`
   z-index: 2;
   display: flex;
-  height: 52px;
   align-items: center;
   justify-content: space-between;
+  @media ${mediaQueries.sm} {
+    flex-direction: column;
+    justify-content: stretch;
+    align-items: flex-start;
+  }
 `;
+// const HeaderContainer = styled.div`
+//   z-index: 2;
+//   display: flex;
+//   align-items: center;
+//   justify-content: space-between;
+
+//   margin: 0 auto;
+//   max-width: 980px;
+//   padding: 0 22px;
+//   padding-left: calc(max(22px, env(safe-area-inset-left)));
+//   padding-right: calc(max(22px, env(safe-area-inset-right)));
+//   position: relative;
+//   z-index: 2;
+//   @media ${mediaQueries.sm} {
+//     flex-direction: column;
+//     justify-content: stretch;
+//     align-items: flex-start;
+//   }
+// `;
```

Comment:
Is this commented section needed for the future? Otherwise it can/should be removed imho.